packages/admin/cms-admin/src/generator/future/generator.ts

```diff
@@ -35,6 +36,14 @@ type MultiFileFormFieldConfig = { type: "fileUpload"; multiple: true; maxFiles?:
 export type FormFieldConfig<T> = (
     | { type: "text"; multiline?: boolean }
     | { type: "number" }
+    | {
+          type: "numberRange";
+          minValue: number;
+          maxValue: number;
+          disableSlider?: boolean;
+          startAdornment?: ReactNode;
```

Comment:
You can not use abritary react components in generator config, please allow only string

Reply:
I can think of other use cases for the adornment, e.g. icons. But we can add this later. String is good for now.